packages/ckeditor5-bookmark/src/bookmarkui.ts

```diff
@@ -251,7 +251,10 @@ export default class BookmarkUI extends Plugin {
 				id: bookmarkId,
 				label: bookmarkId,
 				href: `#${ bookmarkId }`,
-				icon: icons.bookmark
+				icon: icons.bookmark,
+				preview: {
+					tooltip: t( 'Scroll to bookmark' )
+				}
```

Comment:
I'd flatten this as `linkPreviewTooltip` as only there we do need this customization.

Reply:
Currently, we cannot do that. I discovered that preview used a different icon that list item, so I'd keep it as it is to avoid repetition of prefixes.